docs/upgrade_from_k8s_12.md

```diff
@@ -1,191 +1 @@
-    --filter Name=tag:KubernetesCluster,Values=${NEW_NAME} \
-             Name=tag-key,Values=k8s.io/role/master \
-             Name=instance-state-name,Values=running \
-    --query Reservations[].Instances[].PublicIpAddress \
-    --output text
-```
-
-Then `ssh admin@<ip>` (the SSH key will be the one you added above, i.e. `~/.ssh/id_rsa.pub`), and run:
-
-First check that the apiserver is running:
-```
-kubectl get nodes
-```
-
-You should see only one node (the master).  Then run
-```
-NS=`kubectl get namespaces -o 'jsonpath={.items[*].metadata.name}'`
-for i in ${NS}; do kubectl get secrets --namespace=${i} --no-headers | grep "kubernetes.io/service-account-token" | awk '{print $1}' | xargs -I {} kubectl delete secret --namespace=$i {}; done
-sleep 60 # Allow for new secrets to be created
-kubectl delete pods -lk8s-app=dns-controller --namespace=kube-system
-kubectl delete pods -lk8s-app=kube-dns --namespace=kube-system
-```
-
-
-You probably also want to delete the imported DNS services from prior versions:
-
-```
-kubectl delete rc -lk8s-app=kube-dns --namespace=kube-system
-```
-
-
-Within a few minutes the new cluster should be running.
-
-Try `kubectl get nodes --show-labels`, `kubectl get pods --all-namespaces` etc until you are sure that all is well.
-
-This should work even without being SSH-ed into the master, although it can take a few minutes
-for DNS to propagate.  If it doesn't work, double-check that you have specified a valid
-domain name for your cluster, that records have been created in Route53, and that you
-can resolve those records from your machine (using `nslookup` or `dig`).
-
-## Other fixes
-
-* If you're using a manually created ELB, the auto-scaling groups change, so you will need to reconfigure
-your ELBs to include the new auto-scaling group(s).
-
-* It is recommended to delete old kubernetes system services that we imported (and replace them with newer versions):
-
-```
-kubectl delete rc -lk8s-app=kube-dns --namespace=kube-system
-
-kubectl delete rc -lk8s-app=elasticsearch-logging --namespace=kube-system
-kubectl delete rc -lk8s-app=kibana-logging --namespace=kube-system
-kubectl delete rc -lk8s-app=kubernetes-dashboard --namespace=kube-system
-kubectl delete rc -lk8s-app=influxGrafana --namespace=kube-system
-
-kubectl delete deployment -lk8s-app=heapster --namespace=kube-system
-```
-
-## Delete remaining resources of the old cluster
-
-`kops delete cluster ${OLD_NAME}`
-> ```
-TYPE                    NAME                                    ID
-autoscaling-config      kubernetes-minion-group-us-west-2a      kubernetes-minion-group-us-west-2a
-autoscaling-group       kubernetes-minion                       kubernetes-minion-group-us-west-2a
-instance                kubernetes-master                       i-67af2ec8
-```
-
-And once you've confirmed it looks right, run with `--yes`
-
-You will also need to release the old ElasticIP manually.
+[Moved here](upgrade_from_kubeup.md)
```

Comment:
Not dumb question, me being dumb and forgetting to add the file.  Just added it - thank you for catching :-)